src/pybind/mgr/cephadm/serve.py

```diff
@@ -575,6 +570,33 @@ def _check_for_strays(self) -> None:
                 }
         self.mgr.set_health_checks(self.mgr.health_checks)
 
+    def _check_for_moved_osds(self) -> None:
```

Comment:
Do we want to add a call to this as part of the actual serve loop? As far as I can see right now this function isn't being run outside of the tests

Reply:
whoops!